### PR TITLE
Azure DevOps Pipeline improvements

### PR DESCRIPTION
**Pull request by ypc-faros**

### CODE REVIEW

File: sources/azurepipeline-source/resources/spec.json [hunk 1]
```diff
@@ -7,41 +7,59 @@
     "required": [
       "access_token",
       "organization",
-      "project_names",
-      "cutoff_days"
+      "project_names"
```
Comment (cjwooo) [inline sources/azurepipeline-source/resources/spec.json:10]: You renamed this config to `projects`. Also, this rename will require an Alastor update as well, as its corresponding catalog entry currently refers to `project_names`.

File: destinations/airbyte-faros-destination/src/converters/azurepipeline/common.ts [hunk 2]
```diff
@@ -68,6 +79,29 @@ export abstract class AzurePipelineConverter extends Converter {
...
+
+    const parts = repo.id.split('/');
+    // Expecting repo.id to be in the format of <org>/<repo>
+    // E.g., faros-ai/airbyte-connectors
+    if (parts.length < 2) {
+      return undefined;
+    }
+
+    return {
+      name: parts[1],
```
Comment (cjwooo) [inline destinations/airbyte-faros-destination/src/converters/azurepipeline/common.ts:96]: Lowercase the `vcs_Repository.name` and `vcs_Organization.uid` values to match behavior of other connectors. Also, `name` is not part of `vcs_Organization`'s key.

File: sources/azurepipeline-source/resources/spec.json (hunk 1 above)
Comment (ypc-faros) [reply sources/azurepipeline-source/resources/spec.json:10]: Yes I have other changes in mind for the catalog. I'll include this one as well, thank you for the heads up 🙏

**Review by cjwooo — APPROVED** [review]
Just a couple comments

**Pull request merged**